spec/policies/course_lesson_part_policy_spec.rb

```diff
@@ -12,8 +12,18 @@
     it { is_expected.to permit_edit_and_update_actions }
     it { is_expected.to permit_action(:show_split) }
     it { is_expected.to permit_action(:split) }
-    it { is_expected.to permit_action(:show_delete) }
-    it { is_expected.to permit_action(:destroy) }
+    it { is_expected.to forbid_action(:show_delete) }
+    it { is_expected.to forbid_action(:destroy) }
+
+    it "permits the show_delete action for admin when there is > 1 course lesson part" do
+      FactoryBot.create(:course_lesson_part, course_lesson: course_lesson_part.course_lesson)
+      is_expected.to permit_action(:show_delete)
+    end
+
+    it "permits the destroy action for admin when there is > 1 course lesson part" do
+      FactoryBot.create(:course_lesson_part, course_lesson: course_lesson_part.course_lesson)
+      is_expected.to permit_action(:destroy)
+    end
```

Comment:
what about when there isn't more than 1 lesson part? :)

Reply:
Would the tests on lines 15 & 16? cover that. I've not explicit said in the it statement, it's just that it forbids it. Happy to put it in an it block with a description if that'd make it easier to read.

Reply:
Ah, you are right, they technically cover it.

Yes, I think a description would be very useful.